arcaflow_plugin_metadata/metadata_plugin.py

```diff
@@ -49,7 +56,7 @@ def collect_metadata(
         return "success", selected_facts_schema.unserialize(output)
 
     except KeyError:
-        return "error", ErrorOutput("missing a key in ansible facts")
+        return "error", ErrorOutput("Missing a key in ansible facts")
```

Comment:
Should the message mention _which_ key is missing?  E.g.,
```suggestion
    except KeyError as exc:
        return "error", ErrorOutput(f"Missing a key in ansible facts: {exc}")
```